servers/quarkus-server/src/test/java/org/projectnessie/server/RestGitTest.java

```diff
@@ -65,9 +67,9 @@ public void testBasic() {
     Assertions.assertEquals(preSize + 1, references.length);
 
     Reference reference = rest().get("trees/tree/mainx").then()
-                 .statusCode(200)
-                 .extract()
-                 .as(Reference.class);
+        .statusCode(200)
```

Comment:
nit: lots of whitespace changes in this test.

Reply:
this is coming from the formatter unfortunately :(